tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -266,71 +185,80 @@ def _get_support_ldpmode_physical_ports(
             power_class_docker_cmd = asichost.get_docker_cmd(power_class_cmd, "database")
             power_class = duthost.command(power_class_docker_cmd)["stdout"]
 
-            if "QSFP" not in sfp_type or "Power Class 1" in power_class:
+            if ("QSFP" not in sfp_type and "OSFP" not in sfp_type) or "Power Class 1" in power_class:
                 logging.info("skip testing port {} which doesn't support LPM".format(intf))
+                not_supporting_lpm_physical_ports.add(phy_intf)
                 continue
-            tested_lpmode_physical_ports.add(phy_intf)
-            tested_lpmode_ports.add(intf)
-            if intf in original_interface_status and original_interface_status[intf]["admin"].lower() == "up":
-                tested_lpmode_ports_with_admin_up.add(intf)
+            tested_physical_ports.add(phy_intf)
+            logging.info("setting {} physical interface {}".format(intf, phy_intf))
+            new_lpmode = "off" if original_lpmode[intf].lower() == "on" else "on"
+            lpmode_set_result = duthost.command("{} {} {}".format(cmd_sfp_set_lpmode, new_lpmode, intf))
+            assert lpmode_set_result["rc"] == 0, "'{} {} {}' failed".format(cmd_sfp_set_lpmode, new_lpmode, intf)
+    time.sleep(10)
```

Comment:
why sleep and not polling?

Reply:
This is the old implementation, I just revert it to the old one.